src/chronos.erl

```diff
@@ -78,9 +78,8 @@ stop_timer(ServerName, TimerName) ->
 %%%===================================================================
 
 
--spec init(server_name()) -> {'ok', #chronos_state{}} | {'stop', term()} | 'ignore'.
-init(ServerName) ->
-    gproc:reg(proc_name(ServerName)),
+-spec init([]) -> {'ok', #chronos_state{}} | {'stop', term()} | 'ignore'.
```

Comment:
The spec in line 81 uses a record, please define a type for the record and use that instead.